src/migration/1699508350425-UserAlertsTrigger.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserAlertsTrigger1699508350425 implements MigrationInterface {
+    name = 'UserAlertsTrigger1699508350425'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+        CREATE OR REPLACE FUNCTION insert_user_alerts()
+          RETURNS TRIGGER
+          LANGUAGE PLPGSQL
+          AS
+        $$
+        BEGIN
+          INSERT INTO public.alerts ("userId") VALUES (NEW."id");
+          RETURN NEW;
+        END;
+        $$
+      `)
+    queryRunner.query('CREATE TRIGGER user_create_trigger AFTER INSERT ON public."user" FOR EACH ROW EXECUTE PROCEDURE insert_user_alerts()')
```

Comment:
Let's give this trigger a more explicit name. We may add more triggers in the future.

Reply:
Right. Let me update the naming